bluebrain/repo-bluebrain/packages/py-mvdtool/package.py

```diff
@@ -42,10 +42,3 @@ class PyMvdtool(PythonPackage):
     depends_on('libsonata+mpi', type=('build', 'run'), when='+mpi@2.1:')
     depends_on('highfive~mpi', type='build', when='~mpi')
     depends_on('highfive+mpi', type='build', when='+mpi')
-
-    @run_before('build')
-    def configure(self):
-        # we cant use @when('+mpi'), raises NoSuchMethodError
-        if self.spec.satisfies('+mpi'):
-            env['CC'] = self.spec['mpi'].mpicc
-            env['CXX'] = self.spec['mpi'].mpicxx
```

Comment:
I assume this is no longer required...

Reply:
Future Spack has different conventions for Python packages :)